### PR TITLE
Giovana np1 patch 1

### PR DESCRIPTION
**Pull request by GiovanaNp1**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by GiovanaNp1** [comment]
@davorpa  Please, you can reviews my PR again?

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -393,6 +394,18 @@
...
+### Elastic Stack
+
+* [App Search Fundamentals](https://www.elastic.co/pt/training/app-search-fundamentals) - Elastic .co
+* [App Search Web Crawler Fundamentals](https://www.elastic.co/pt/training/app-search-web-crawler-fundamentals) - Elastic .co
+* [ECE Fundamentals](https://www.elastic.co/pt/training/ece-fundamentals) - Elastic .co
+* [Elastic Security Fundamentals: SIEM]([https://www.elastic.co/pt/training/kibana-for-splunk-spl-users](https://www.elastic.co/pt/training/elastic-security-fundamentals-siem)) - Elastic .co
+* [Kibana for Splunk SPL Users](https://www.elastic.co/pt/training/kibana-for-splunk-spl-users) - Elastic .co
+* [Kibana Fundamentals](https://www.elastic.co/pt/training/kibana-fundamentals) - Elastic .co
+* [Observability Fundamentals](https://www.elastic.co/pt/training/observability-fundamentals) - Elastic .co
+* [Workplace Search Fundamentals](https://www.elastic.co/pt/training/workplace-search-fundamentals) - Elastic .co
```
Comment (davorpa) [inline courses/free-courses-en.md:406]: Removing country path from url :wink:

```suggestion
* [App Search Fundamentals](https://www.elastic.co/training/app-search-fundamentals) - Elastic .co
* [App Search Web Crawler Fundamentals](https://www.elastic.co/training/app-search-web-crawler-fundamentals) - Elastic .co
* [ECE Fundamentals](https://www.elastic.co/training/ece-fundamentals) - Elastic .co
* [Elastic Security Fundamentals: SIEM]([https://www.elastic.co/training/kibana-for-splunk-spl-users](https://www.elastic.co/training/elastic-security-fundamentals-siem)) - Elastic .co
* [Kibana for Splunk SPL Users](https://www.elastic.co/training/kibana-for-splunk-spl-users) - Elastic .co
* [Kibana Fundamentals](https://www.elastic.co/training/kibana-fundamentals) - Elastic .co
* [Observability Fundamentals](https://www.elastic.co/training/observability-fundamentals) - Elastic .co
* [Workplace Search Fundamentals](https://www.elastic.co/training/workplace-search-fundamentals) - Elastic .co
```

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -26,6 +26,7 @@
 * [Deep Learning](#deep-learning)
 * [DevOps](#devops)
 * [Docker](#docker)
+* [Elastic Stack](#elastic-stack)
```
Comment (eshellman) [inline courses/free-courses-en.md:29]: Our lists use "Elastic" and "Elasticsearch" as categories, and the books-subjects list puts Elastic search under "search Engines" we also have a Kibana resource under a generic heading. Maybe easier to add the elastic stack resources in a separate PR that organizes Elastic resources coherently. Also, do you think Kibana and ElasticSearch really belong in the same category? or in an Elastic.co category off to themselves?

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -393,6 +394,18 @@
 * [IIEC RISE 1.0 Docker](https://www.youtube.com/playlist?list=PLAi9X1uG6jZ30QGz7FZ55A27jPeY8EwkE)
 
 
+### Elastic Stack
+
+* [App Search Fundamentals](https://www.elastic.co/training/app-search-fundamentals) - Elastic .co
```
Comment (eshellman) [inline courses/free-courses-en.md:399]: why the space between elastic and '.co'?

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**